flare/src/main/AndroidManifest.xml

```diff
@@ -6,4 +6,10 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <application>
+        <provider
+            android:authorities="io.pantheon.flare.FlareProvider"
+            android:name=".FlareProvider"
+            android:exported="false" />
+    </application>
 </manifest>
```

Comment:
change this name to flare context provider

Reply:
Done